src/main/java/rife/authentication/elements/Authenticated.java

```diff
@@ -67,8 +67,7 @@ protected void initializeAuthenticated(Context c) {
     protected void sessionNotValid(String authCookieName, String authCookieValue, int validityId) {
     }
 
-    public void process(Context c)
-    throws Exception {
+    public void process(Context c) {
```

Comment:
I think this might prevent classes that extend this class from throwing exceptions

Reply:
It shouldn't. I have extended bld methods adding a throws to the declarations.